README.md

```diff
@@ -96,9 +96,9 @@ for the [Maps](https://github.com/JeroenDeDauw/Maps) and
 
 * The parsers no longer extend `StringValueParser`
 	* They no longer have public methods `setOptions` and `getOptions`
-	* With a few exceptions, they no longer have protected field `options`
-	* With a few exceptions, they no longer have protected methods `getOption`, `requireOption`,
-	  `defaultOption` and `stringParse`
+	* They no longer have protected field `options`
+	* They no longer have protected methods `requireOption`, `defaultOption` and `stringParse`
+	* Except for the `LatLongParserBase` derivatives, they no longer have protected method `getOption`
```

Comment:
Something is off with this sentence. LatLongParserBase actually does have a protected getOption. The way this sentence is currently constructed makes it sound like LatLongParserBase is an exception in so far that it is the only one where this method got removed, when it is the other way around.

Even more confusing is the fact that basically all parsers extend `LatLongParserBase`, except for two. So why not list them here?

* `GlobeCoordinateParser` and `LatLongParser` no longer have protected method `getOption`.

Reply:
Just go for it